src/main/java/lotto/model/LottoPrize.java

```diff
@@ -0,0 +1,35 @@
+package lotto.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum LottoPrize {
+    FIRST(6, 2000000000),
```

Comment:
언더 스코어를 활용해 가독성을 높여도 좋을 것 같아요.
```suggestion
    FIRST(6, 2_000_000_000),
```